i18n/en.json

```diff
@@ -3722,7 +3722,7 @@
   "sidebar_left.channel_navigator.channelSwitcherLabel": "Channel Switcher",
   "sidebar_left.channel_navigator.goBackLabel": "Back",
   "sidebar_left.channel_navigator.goForwardLabel": "Forward",
-  "sidebar_left.channel_navigator.jumpTo": "Jump to...",
+  "sidebar_left.channel_navigator.jumpTo": "Find a channel...",
```

Comment:
Let's change to "Find channel" - remove the 'a' and the '...' - it doesn't fit well when you hover and the shortcut shows